types/NewWindow.d.ts

```diff
@@ -13,6 +13,14 @@ declare module 'react-new-window' {
   export interface IWindowFeatures {
     height: number
     width: number
+    left: number
+    top: number
+    menubar: boolean
+    toolbar: boolean
+    scrollbars: boolean
+    location: boolean
+    status: boolean
+    resizable: boolean
```

Comment:
It seems valid window feature [are limited](https://developer.mozilla.org/en-US/docs/Web/API/Window/open#parameters) to: `popup`, `width`, `height`, `left`, `top`, `noopener` and `noreferrer`. The others are no longer working in modern browser.